src/lang/KclSingleton.ts

```diff
@@ -386,24 +379,11 @@ function defaultSelectionFilter(
   ) as SketchGroup | ExtrudeGroup
   firstSketchOrExtrudeGroup &&
     engineCommandManager.sendSceneCommand({
-      type: 'modeling_cmd_batch_req',
-      batch_id: uuidv4(),
-      responses: false,
-      requests: [
-        {
-          cmd_id: uuidv4(),
-          cmd: {
-            type: 'edit_mode_enter',
-            target: firstSketchOrExtrudeGroup.id,
-          },
-        },
-        {
-          cmd_id: uuidv4(),
-          cmd: {
-            type: 'set_selection_filter',
-            filter: ['face', 'edge', 'solid2d'],
-          },
-        },
-      ],
+      type: 'modeling_cmd_req',
+      cmd_id: uuidv4(),
+      cmd: {
+        type: 'set_selection_filter',
+        filter: ['face', 'edge', 'solid2d', 'curve'],
```

Comment:
adding curve here makes sketch segments selectable from outside of sketch mode. There's a chance this could break something, though I doubt it, and couldn't find anything from some quick tests. We have enough tests around selections that I'm confident they will find something if there's an issue.